### DISCUSSION TITLE
redis 5.0功能

### DISCUSSION THREAD

**Discussion by gitzhuyongfu** [hhyo/Archery#2076] [Ideas]
### 重现步骤

问题：
1.什么时候支持redis5.0以上的功能？
2.什么时候可以支持stream类型的查询？

### 预期外的结果

希望能提上日程

### 版本

1.9.1

### 部署方式

Docker

### 是否还有其他可以辅助定位问题的信息？比如数据库版本等

docker部署

**Comment by LeoQuote** [comment]
你好，欢迎pr

**Comment by batmanjo** [comment]
这个问题解决了吗？
我的代码使用了redis stream 相关的功能
